FLIR/conservator/run_tasks_plugins/conservator_cli.py

```diff
@@ -0,0 +1,11 @@
+from FLIR.conservator.cli import main
+
+
+def conservator_cli(cmd_class, cmd_name, args=(), log="INFO", task=None):
+    cli_cmd = [cmd_class, cmd_name] + list(args)
+    result = main(cli_cmd, standalone_mode=False)
+    print("RESULT:", result)
```

Comment:
Is printing this ok/typical behavior?

Reply:
It's ok, there is some precedence for using print() in other plugins